### ISSUE TITLE
Add additional languages and say which ones are available.

### ISSUE THREAD

**Issue by Jourdelune** [mozilla/firefox-translations#388]
**Is your feature request related to a problem? Please describe.**
When using this extension, I wanted to translate sites from English to French, only it did not translate. I then realized by searching the github that the language I was trying to translate to was not yet available (although it is stated that it will be). 

**Describe the solution you'd like**
I think that the right behaviour would be to indicate that the language present in the page is not yet supported, at the risk of believing that the extension does not work and thus putting a bad mark on the extension (whereas this judgement is false). 

**Describe alternatives you've considered**
I think that creating a navbar at the top of the page is a good idea, but there is some problem that I faced: It is not possible to close the extension's navbar if the site has been translated (which is a bit disturbing, the extension should be discrete). The output language is not specified (I think the output language is based on the browser language, which is not a bad thing but brings confusion when the language is not supported). So it would be nice for example to change the background of the navbar to red when the language is not supported. Finally it would be useful if we could also specify the output language for the extension, it would be clearer.

**Additional context**
There are quite few languages supported, although each one is well trained (the translation quality is good). It would be nice to train the translation models for the 100 most popular languages (or 104 like m2m100 from facebook) into English. This would allow to translate in any language, indeed one could translate a text from French to English and then from English to German, there would need 200 models and not 10 000 to cover all languages).

**Comment by andrenatal** [comment]
Hi @Jourdelune . French is being trained by the consortium of universities as part of the [bergamot project](https://browser.mt/) and is at least expected to be available at some point in June

**Comment by Jourdelune** [comment]
yes indeed, I saw it here: https://github.com/mozilla/firefox-translations-models/. Is it planned to add more languages in the future? And if so which one?

**Comment by Jourdelune** [comment]
> hi @andrenatal, yes indeed, I saw it here: https://github.com/mozilla/firefox-translations-models/. Is it planned to add more languages in the future? And if so which one?

The answer is here: https://github.com/mozilla/firefox-translations/discussions/67#discussioncomment-2090952. I hope that the EU will accept this finencement!

**Comment by andrenatal** [comment]
These were fixed.
